project/scripts/bootstrapCmdTests

```diff
@@ -83,3 +83,12 @@ clear_out "$OUT"
 ./bin/scalac -d "$OUT/out.jar" tests/pos/i12973.scala
 echo "Bug12973().check" | TERM=dumb ./bin/scala -cp "$OUT/out.jar" > "$tmp" 2>&1
 grep -qe "Bug12973 is fixed" "$tmp"
+
+echo "testing -sourcepath with inlining"
```

Comment:
Add a comment explaining what this is testing exactly / why it can't be tested with an sbt scripted test?